sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/DataSourceReadBenchmark.scala

```diff
@@ -154,6 +154,16 @@ object DataSourceReadBenchmark extends SqlBasedBenchmark {
           }
         }
 
+        sqlBenchmark.addCase("SQL Parquet Vectorized (Delta Binary)") { _ =>
+          spark.sql(s"select $query from parquetV2Table").noop()
+        }
+
+        sqlBenchmark.addCase("SQL Parquet MR (Delta Binary)") { _ =>
+          withSQLConf(SQLConf.PARQUET_VECTORIZED_READER_ENABLED.key -> "false") {
+            spark.sql(s"select $query from parquetV2Table").noop()
```

Comment:
Not in this PR, but the table name looks like a general parquet table. If this is particular for delta binary, it seems better to have a specified table name.

Reply:
Actually, I think I will rename the case instead of the table. Instead of `SQL Parquet MR (Delta Binary)` make it `SQL Parquet V2 MR`. I can do that in a subsequent PR as I add the remaining delta encodings.

Reply:
Maybe we should compare every scene for Parquet data page V1 and Parquet data page V2 in `DataSourceReadBenchmark`, if it is only for the data type 'delta binary',  maybe we should add a new benchmark file

Reply:
Not in this pr, I used GA to run a comparison for parquet V1 and V2. I found that in scenes `Single INT Column` and `Single BIGINT Column`, V2 is obviously slower than V1:

```
OpenJDK 64-Bit Server VM 1.8.0_312-b07 on Linux 5.11.0-1025-azure
Intel(R) Xeon(R) CPU E5-2673 v4 @ 2.30GHz
SQL Single INT Column Scan:               Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
------------------------------------------------------------------------------------------------------------------------
SQL CSV                                           19203          19257          76          0.8        1220.9       1.0X
SQL Json                                          11971          11982          15          1.3         761.1       1.6X
SQL Parquet Vectorized: DataPageV1                  195            201           8         80.7          12.4      98.5X
SQL Parquet MR: DataPageV1                         2665           2680          20          5.9         169.4       7.2X
SQL Parquet Vectorized: DataPageV2                  377            390          17         41.8          23.9      51.0X
SQL Parquet MR: DataPageV2                         2326           2356          43          6.8         147.9       8.3X
SQL ORC Vectorized                                  302            316           9         52.0          19.2      63.5X
SQL ORC MR                                         2168           2178          14          7.3         137.8       8.9X

OpenJDK 64-Bit Server VM 1.8.0_312-b07 on Linux 5.11.0-1025-azure
Intel(R) Xeon(R) CPU E5-2673 v4 @ 2.30GHz
Parquet Reader Single INT Column Scan:       Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
---------------------------------------------------------------------------------------------------------------------------
ParquetReader Vectorized: DataPageV1                   289            296           7         54.4          18.4       1.0X
ParquetReader Vectorized -> Row: DataPageV1            283            292           9         55.6          18.0       1.0X
ParquetReader Vectorized: DataPageV2                   460            470           7         34.2          29.2       0.6X
ParquetReader Vectorized -> Row: DataPageV2            461            474          15         34.1          29.3       0.6X

OpenJDK 64-Bit Server VM 1.8.0_312-b07 on Linux 5.11.0-1025-azure
Intel(R) Xeon(R) CPU E5-2673 v4 @ 2.30GHz
SQL Single BIGINT Column Scan:            Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
------------------------------------------------------------------------------------------------------------------------
SQL CSV                                           24759          24790          45          0.6        1574.1       1.0X
SQL Json                                          15345          15393          67          1.0         975.6       1.6X
SQL Parquet Vectorized: DataPageV1                  304            314          12         51.8          19.3      81.5X
SQL Parquet MR: DataPageV1                         2798           2824          38          5.6         177.9       8.9X
SQL Parquet Vectorized: DataPageV2                  600            618          24         26.2          38.2      41.2X
SQL Parquet MR: DataPageV2                         2423           2450          38          6.5         154.1      10.2X
SQL ORC Vectorized                                  376            381           3         41.9          23.9      65.9X
SQL ORC MR                                         2245           2260          21          7.0         142.7      11.0X

OpenJDK 64-Bit Server VM 1.8.0_312-b07 on Linux 5.11.0-1025-azure
Intel(R) Xeon(R) CPU E5-2673 v4 @ 2.30GHz
Parquet Reader Single BIGINT Column Scan:    Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
---------------------------------------------------------------------------------------------------------------------------
ParquetReader Vectorized: DataPageV1                   372            423          96         42.2          23.7       1.0X
ParquetReader Vectorized -> Row: DataPageV1            376            382           7         41.8          23.9       1.0X
ParquetReader Vectorized: DataPageV2                   678            694          23         23.2          43.1       0.5X
ParquetReader Vectorized -> Row: DataPageV2            673            686          13         23.4          42.8       0.6X
```